modules/ixBidAdapter.js

```diff
@@ -416,7 +416,11 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
       msd = impressions[transactionIds[i]].missingCount;
     }
 
-    trimImpressions(impressions[transactionIds[i]], MAX_REQ_SIZE - BASE_REQ_SIZE);
+    if (BASE_REQ_SIZE < MAX_REQ_SIZE) {
```

Comment:
Is this backwards? Trim if req > max ?

Reply:
We want to make sure that the `BASE_REQ_SIZE` created above isn't larger than the `MAX_REQ_SIZE`, this fix is to ensure that we don't pass a [negative value to `trimImpressions` function](https://github.com/prebid/Prebid.js/pull/6582/files#diff-2597e133de10cd404cf919d82b772fa4a4ba33e49e9f4bdc4ba18a0a905774a5R559).